docker/fetch_normalization_data.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+"""
+Fetches normalization data.
+
+Usage:
+
+    python docker/fetch_normalization_data.py
+
+"""
+
+import argparse
+import datetime
+import os
+import sys
+
+import psycopg2
+import requests
+from six.moves.urllib.parse import quote_plus
+
+
+# NOTE(willkg): These platforms come from the -prod data.
+PLATFORMS = ['Mac OS X', 'Linux', 'Windows', 'Unknown']
+
+
+def fetch_data_from_api(endpoint, product, version, platform, start_date, end_date):
+    """Fetches data from API"""
+    params = []
+    params.append(('end_date', end_date))
+    params.append(('start_date', start_date))
+    params.append(('product', product))
+    params.append(('platforms', platform))
+    params.append(('versions', version))
+
+    qs = '&'.join(['%s=%s' % (param[0], quote_plus(param[1])) for param in params])
+    url = endpoint + '?' + qs
+    req = requests.get(url)
```

Comment:
Oo--I can redo this params stuff now that I'm not passing in multiples of things. I'll do that now.